Makefile

```
@@ -302,8 +259,8 @@ gcs-publish-ci: gsutil version-dist-ci
 	echo "VERSION: ${VERSION}"
 	echo "${GCS_URL}/${VERSION}" > ${UPLOAD}/${LATEST_FILE}
 	gsutil -h "Cache-Control:private, max-age=0, no-transform" cp ${UPLOAD}/${LATEST_FILE} ${GCS_LOCATION}
-	mkdir -p ${BAZEL_BIN}/cmd/kops/linux-amd64/
-	cp  ${DIST}/linux/amd64/kops ${BAZEL_BIN}/cmd/kops/linux-amd64/
+	mkdir -p .bazel-bin/cmd/kops/linux-amd64/
+	cp  ${DIST}/linux/amd64/kops .bazel-bin/cmd/kops/linux-amd64/
```

Maybe this will fix the build:
```suggestion
	mkdir -p bazel-bin/cmd/kops/linux-amd64/
	cp  ${DIST}/linux/amd64/kops bazel-bin/cmd/kops/linux-amd64/
```

---

Yep. I think so